swagger.yaml

```diff
@@ -8,6 +8,58 @@ security:
   - bearerAuth: []
 
 paths:
+  "/users/{userId}/vacation":
+    put:
+      operationId: updateUserVacation
+      summary: Update user vacation days
+      description: Update vacation days for a specific user by year.
+      tags:
+        - Update vacation
+      parameters:
+        - name: userId
+          in: path
+          required: true
+          description: The ID of the user to update vacation for
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                vacationDays:
+                  type: object
+                  additionalProperties:
+                    type: object
+                    properties:
+                      total:
+                        type: integer
+                      remaining:
+                        type: integer
+                  description: Vacation days by year
+              required:
+                - vacationDays
+      responses:
+        "200":
+          description: Successfully updated the vacation days
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  id:
+                    type: string
+                    description: The ID of the updated user
+                  updatedFields:
+                    type: array
+                    items:
+                      type: string
+                    description: List of updated fields (e.g., `vacationDaysByYear`, `unspentVacationDaysByYear`, etc.)
```

Comment:
Wouldn't it be better to just return the updated attribute and its updated value e.g. vacation days:
{
  total: 30,
  remaining: 15
}

Rather than a list of strings saying what has been updated but not showing how it's been updated.
Think this would be more typical REST conventions

Reply:
So could just return vacation days